vagrant/config/docker/config.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
I think multi config maybe more proper, in consideration of the possible big difference between docker/ova/vagrant test stack in the future.